Packs/FreshworksFreshservice/Integrations/FreshworksFreshservice/FreshworksFreshservice.py

```diff
@@ -0,0 +1,3630 @@
+                args.get('requester_id'),
+                args.get('phone'),
+                args.get('email')
+            ])
+    ]):
+        raise ValueError(
+            'One of the following is mandatory: requester_id, phone, email')
+
+
+def get_files_to_attach(args: Dict[str, Any], ) -> List[Tuple]:
+    """ Get array of files to attach in Freshservice.
+
+    Args:
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        List[Tuple]: List of files.
+    """
+    return [get_file(file) for file in argToList(args.get('attachments'))]
+
+
+def convert_response_properties(
+    updated_response: List[dict] | Dict[str, Any],
+    predefined_values: Dict[str, Dict[str, int]],
+) -> List[dict] | Dict[str, Any]:
+    """ Convert command properties from number to string for
+        the XSOAR output.
+
+    Args:
+        updated_response (List[dict]): Freshservice response.
+        predefined_values (Dict[str, Dict[str, int]]): Predefined values for Freshservice.
+
+    Returns:
+        List[dict]: Updated response.
+    """
+    if not updated_response:
+        return updated_response
+
+    if isinstance(updated_response, dict):
+        updated_response = [updated_response]
+
+    response_keys = updated_response[0].keys()
+
+    for item in updated_response:
+        for response_key in response_keys:
+            if item.get(
+                    response_key) and response_key in predefined_values.keys():
+
+                property_values = reverse_dict(predefined_values[response_key])
+                item[response_key] = property_values.get(item[response_key])
+
+    return updated_response
+
+
+def reverse_dict(dict_: Dict[str, Any]) -> Dict[str, Any]:
+    """ Reverse dictionary.
+    Args:
+        dict_ (dict[str, Any]): Dictionary to reverse.
+    Returns:
+        dict[Any, str]: Reversed dictionary.
+    """
+    return {v: k for k, v in dict_.items()}
+
+
+def get_args_by_command_name(args: Dict[str, Any]) -> Tuple:
+    """ Return the default command args by the command name.
+
+    Args:
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        Tuple: The arguments.
+    """
+    command_name = args['command_name'].split("-")
+    command_operator = command_name[-1]
+    entity_name = command_name[1]
+    entity_id_key = f'{entity_name}_id'
+    output_prefix = entity_name.capitalize()
+
+    # checks if command entity consists of two words.
+    if (len(command_name) > MAX_DEFAULT_ARGS_COUNT
+            and not {'task', 'conversation'}.intersection(command_name)):
+        entity_id_key = f'{entity_name}_{command_name[2]}_id'
+        entity_name = f'{entity_name}_{command_name[2]}'
+        output_prefix = f'{output_prefix}{command_name[2].capitalize()}'
+
+    return args.get(
+        entity_id_key), entity_name, output_prefix, command_operator
+
+
+def get_default_seconds(time_in_seconds: int) -> int:
```

Comment:
We don't need to use a function for one line, especially if we only use it ones

Reply:
Same as before, but why would we want to repeat a logic multiple times? Also, it will be easier to read and maintain if any changes are needed.